Sources/Screens/Home/TabbedViewModel.swift

```diff
@@ -0,0 +1,35 @@
+import GDSCommon
+import UIKit
+
+final class TabbedViewModel: NSObject, BaseViewModel {
```

Comment:
Is there a reason this is a class rather than a struct and subclassing `NSObject`?

Reply:
Originally I had this acting as the UITableViewDataSource, but that became unweildy so refactored and forgot to undo this.  Fixed.